recipes/imutils-cpp/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
```

Comment:
```suggestion
        if not cross_building(self):
```
V2 linter will complain. It does not allow tools.cross_building anymore.

Reply:
Thank you for your fix.
I see.